src/analysis/specific_limits.lean

```diff
@@ -620,7 +620,11 @@ end le_geometric
 
 section summable_le_geometric
 
-variables [normed_group α] {r C : ℝ} {f : ℕ → α}
+variables [semi_normed_group α] {r C : ℝ} {f : ℕ → α}
+
+lemma normed_group.cauchy_seq_of_le_geometric {C : ℝ} {r : ℝ} (hr : r < 1)
```

Comment:
This lemma and the next ones are called `normed_group.something`, but they are all dealing with semi_normed_groups. I would just remove the namespace when there is no ambiguity.